pdc/apps/release/migrations/0002_auto_20150512_0719.py

```diff
@@ -53,6 +53,7 @@ def create_variant_types(apps, schema_editor):
         'optional',
         'addon',
         'layered-product',
+        'module',
```

Comment:
@sochotnicky Good catch! We certainly should add our own migration step for that. I'm not sure how that goes with Django, but it can't be that complicated, or very different from how sqlalchemy/alembic do it.